apps/web/pages/account/Security.tsx

```diff
@@ -0,0 +1,63 @@
+import {Button, CircularProgress, Input} from '@usevenice/ui'
+import {EditIcon} from '@usevenice/ui/icons'
+import {useForm} from 'react-hook-form'
+
+interface SecurityFormData {
+  currentPassword: string
+  newPassword: string
+  newPasswordConfirm: string
+}
+
+export function Security() {
+  const {register, handleSubmit, formState} = useForm<SecurityFormData>({
+    reValidateMode: 'onBlur',
+  })
+  return (
+    <form
+      className="flex flex-col items-start gap-6"
+      onSubmit={handleSubmit(async (data) => {
+        await new Promise((resolve) => setTimeout(resolve, 2000))
+        console.log('Successfully update password', data)
```

Comment:
@tonyxiao I think we need an API for this. Seems like supabase only has reset password via Email flow. https://supabase.com/docs/reference/javascript/auth-resetpasswordforemail

@iwasrobbed should we change to that flow instead?

Reply:
Yes, please change to whatever is easiest (& we can even punt on this one for now and just ask people to logout if they need to reset pw for some reason)